Add subscription docs (#74)
ferrocene/criticalup: docs/src/subscriptions.rst

.. SPDX-FileCopyrightText: The Ferrocene Developers
.. SPDX-License-Identifier: MIT OR Apache-2.0

.. _subscriptions:

Managing Subscriptions
======================

This chapter describes how to make changes to your Ferrocene subscription.

> - If you subscribed to Ferrocene using Stripe, you can make changes to your subscription in your `Stripe Billing Portal <https://customers.ferrocene.dev/stripe/billing>`_.

Reviewer: ```suggestion
- If you subscribed to Ferrocene with a credit card, you can make changes to your subscription in your `Billing Portal <https://customers.ferrocene.dev/stripe/billing>`_.
```

Stripe is an implementation detail IMO, it might confuse users who don't know what stripe is, and who just paid with a card.

Reviewer 2: CHecked with @justahero and @BriocheBerlin and this link does work.
